src/App.js

```diff
@@ -81,6 +94,9 @@ class App extends Component {
     });
   };  // End setDev()
 
+  /** Load an individual client's data @techExpertisePlease from what source? Where does toLoad come from?
+   * @param {object} toLoad - An object representing the client data to be loaded @techExpertisePlease does this have a specific shape? Or format (ie is it JSON or something?)
```

Comment:
It's a JSON object that needs to match the client data format.

Reply:
Which can be seen in 'src/utils/CLIENT_DEFAULTS.js', though that could move.

Reply:
Thanks! Updated the JSDoc description for `loadClient` in 269d6dfa6df06d38148aed95d0c563c1dbb3f85b (with even a link to the CLIENT_DEFAULTS that won't break even if it moves! Wow, JSDoc is so fancy! Although this link is to the documentation for the CLIENT_DEFAULTS constant, rather than to the CLIENT_DEFAULTS.js source code, so it will only be as useful as the documentation for CLIENT_DEFAULTS is)

Reply:
Just so you know, there's an `@see` tag in jsdoc, though we're looking for things that will help us actually make the docs visible (talk later) and whatever lib we use may not be that fancy.